### ISSUE TITLE
figure Iñaki suggested

### ISSUE THREAD

**Issue by lizzieinvancouver** [lizzieinvancouver/vin#30]
... for understanding delays:
> 3.- How to represent the differences between varieties:
> - the delay that you observe at veraison, is the combination (accumulation)
> of the delays of all the phenological stages simulated by the model
> (dormancy, budbreak , flowering and veraison). I was thinking that a figure
> we did for a previous paper (Cuccia et al.. - in attached) can be useful to
> understand why we have some delays. Please, see figure 5. This figure allow
> to represent the impact of each temperature in the final calculation of the
> phenological stages. I am also sending you a kind of schema I did in my
> office (sorry for the quality of the picture). The general idea is that for
> two varieties (gre - Grenache in green/chy - chardonnay in blue) we do not
> cover the same range of temperatures (and their associated response
> represented by the model) between the past and the future (Rp - Rate past /
> Rf - Rate future /// FU Forcing Units //// Tpo - minimal value of the
> temperature observed to calculate the phenology in each phase in the past -
> Tp1 maximal value of the temperature observed to calculate the phenology in
> each phase in the past ). This can allow to represent why we have different
> trends between varieties. Let me know if you arrive to understand my schema
>  If we arrive to represent this ranges for each variety and for each
> phase, I think it is possible to understand why we have different trends
> between varieties.

**Comment by lizzieinvancouver** [comment]
As discussed today, first steps are:
(1) Compare stages over time to see which stages are delaying.
(2) Once we know what stages are the issue look at days or hours above the optimal temp for that stage ...
(3) And then try Iñaki's figure for a couple location? (Unless Iñaki has better ideas for a figure that could work across more pixels).

**Comment by MoralesCastilla** [comment]
@lizzieinvancouver 
We have now figure ~GitHub/vin/climatefuture/docs/manuscripts/figures/suppFig_causes_delays_Aust_Spain.png, which should help understanding delays. Since Kees asks about the same issue in his email, I wonder if we should include the figure in the Supp.

**Comment by lizzieinvancouver** [comment]
@MoralesCastilla Yes, sounds good. Once you finalize the caption in the supp and add a ref to it in the main text, please feel free to close this issue!
